README.md

```diff
@@ -37,9 +37,18 @@
 
 <br>
 
-Strapi is a free and open-source headless CMS delivering your content anywhere you need.
 
-- **Keep control over your data**. With Strapi, you know where your data is stored, and you keep full control at all times.
+Strapi is an open-source software helping developers to easily build, deploy, and manage APIs. Its open-source headless CMS provides backend functionality for applications and gives developers the freedom to use the tools of their choice for front-end frameworks. It’s called headless because the frontend is not part of the CMS, but there’s an APIfor content to show up on different devices. Moreover, Strapi is a free and open-source headless CMS delivering your content anywhere you need.
```

Comment:
```suggestion
Strapi is an open-source software helping developers to easily build, deploy, and manage APIs. Its open-source headless CMS provides backend functionality for applications and gives developers the freedom to use the tools of their choice for front-end frameworks. It’s called headless because the frontend is not part of the CMS, but there’s an API for content to show up on different devices. Moreover, Strapi is a free and open-source headless CMS delivering your content anywhere you need.
```